terraform/modules/stack/main.tf

```diff
@@ -1,7 +1,120 @@
-locals {
-  java_opts_heap_size = "-Xss6M -Xms2G -Xmx3G"
+# Ingest service
+
+module "ingest_service" {
+  source = "../service/ingest"
+
+  service_name = "${var.namespace}-ingests-service"
+
+  security_group_ids = [
+    aws_security_group.interservice.id,
+    aws_security_group.service_egress.id
+  ]
+
+  cluster_arn = aws_ecs_cluster.cluster.arn
+
+  external_api_container_image = local.image_ids["ingests_api"]
+  internal_api_container_image = local.image_ids["ingests_tracker"]
```

Comment:
Use the API/tracker names? Let's be consistent.

Reply:
I've removed all the name references and am using this format everywhere as it felt redundant to have a name and a value in the map.